pkg/policystats/policystats.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package policystats
+
+import (
+	"path/filepath"
+
+	"github.com/cilium/ebpf"
+
+	"github.com/cilium/tetragon/pkg/bpf"
+	"github.com/cilium/tetragon/pkg/tracingpolicy"
+)
+
+const (
+	PolicyStatsMapName = "policy_stats"
+)
+
+type PolicyAction uint8
+
+const (
+	// NB: values below should match the ones in bpf/lib/policy_conf.h
+	InvalidAction               PolicyAction = 0
+	PolicyPost                  PolicyAction = 1
+	PolicySignal                PolicyAction = 2
+	PolicyMonitorSignal         PolicyAction = 3
+	PolicyOverride              PolicyAction = 4
+	PolicyMonitorOverride       PolicyAction = 5
+	PolicyNotifyEnforcer        PolicyAction = 6
+	PolicyMonitorNotifyEnforcer PolicyAction = 7
+	PolicyActionsNr                          = 8
+)
+
+type PolicyStats struct {
```

Comment:
Nit: It could make sense to compare struct policy_stats and PolicyStats in [alignchecker](https://github.com/cilium/tetragon/blob/6a6bae661e3d2fb854b4788828dcf3d89e070628/pkg/alignchecker/alignchecker.go#L38). This could help to avoid adding things and forget to update one of the structs.